twinkle.js

```diff
@@ -126,6 +126,9 @@ Twinkle.defaultConfig = {
 	// Hidden preferences
 	autolevelStaleDays: 3, // Huggle is 3, CBNG is 2
 	revertMaxRevisions: 50, // intentionally limited
+	// various hatnote templates, used when tagging
+	// (csd/xfd/tag/prod/protect) to ensure MOS:ORDER
+	hatnoteRegex: 'short description|hatnote|main|correct title|dablink|distinguish|for|further|selfref|year dab|similar names|highway detail hatnote|broader|about(?:-distinguish| other people)?|other\\s?(?:hurricane(?: use)?s|people|persons|places|ships|uses(?: of)?)|redirect(?:-(?:distinguish|synonym|multi))?|see\\s?(?:wiktionary|also(?: if exists)?)',
```

Comment:
This is not a user preference at all. What we're missing is a way to share data/code between modules. It would be better to define this as `Twinkle.hatnoteRegex` in `twinkle.js` file at the bottom under a header called something like: 

```js
/**
 * Data and code shared between multiple modules,
 * but not generic enough to be in morebits.
 */
```

Such a section will also come handy for a lot of other stuff too.

Reply:
Exactly right.  I wrote it that way at first, but shied away from it because there simply wasn't anything else there.  I thought having it in `defaultConfig` might make it more obvious that "this is a thing to change," but, yes, it'd be better to just do it normally.

Reply:
Also a good place for #969